Name: "Just Charge" By Default
Version: 1.3.5-1
Author: Jason Robitaille
Description: On plugging in your device, it'll automatically go into "Just Charge" with notification at bottom if you want to change to a different mode  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -2101,7 +2101,7 @@
 		this.isUSBConnected = true;
 	 	if (!this.onActiveCall) {
 			if (this.canUSBAlertDisplayed()) 
-				this.showStorageModeAlert();
+				this.createUSBDashboard(true);
 			else 
 				this.createUSBDashboard();
 		}
@@ -2235,7 +2235,7 @@
 					if (this.updateAvailable) 
 						this.createUSBDashboard();
 					else 
-						this.showStorageModeAlert();
+						this.createUSBDashboard(true);
 				}							
 			}
 		}.bind(this),	
